emsdk

```diff
@@ -1869,6 +1877,10 @@ def main():
       elif sys.argv[i] == '--disable-assertions':
         ENABLE_LLVM_ASSERTIONS = 'OFF'
         sys.argv[i] = ''
+      elif sys.argv[i] == '--enable-wasm':
+        global ENABLE_WASM
+        ENABLE_WASM = True
+	sys.argv[i] = ''
```

Comment:
The indentation drops here on `sys.argv[i] = ''` by one character. (or perhaps a tabs vs spaces difference?)